internal/output/kafka.go

```diff
@@ -43,7 +43,7 @@ type kafkaConfig struct {
 	KafkaOutputFormat       string        `long:"kafkaoutputformat"           ini-name:"kafkaoutputformat"           env:"DNSMONSTER_KAFKAOUTPUTFORMAT"           default:"json"                                                    description:"Output format. options:json, gob. "                                                                                                                                               choice:"json" choice:"gob"`
 	KafkaTimeout            uint          `long:"kafkatimeout"                ini-name:"kafkatimeout"                env:"DNSMONSTER_KAFKATIMEOUT"                default:"3"                                                       description:"Kafka connection timeout in seconds"`
 	KafkaBatchDelay         time.Duration `long:"kafkabatchdelay"             ini-name:"kafkabatchdelay"             env:"DNSMONSTER_KAFKABATCHDELAY"             default:"1s"                                                      description:"Interval between sending results to Kafka if Batch size is not filled"`
-	KafkaCompress           bool          `long:"kafkacompress"               ini-name:"kafkacompress"               env:"DNSMONSTER_KAFKACOMPRESS"                                                                                 description:"Compress Kafka connection"`
+	KafkaCompress           string        `long:"kafkacompress"               ini-name:"kafkacompress"               env:"DNSMONSTER_KAFKACOMPRESS"               default:""                                                        description:"Compress Kafka connection [snappy, gzip, lz4, zstd]"`
```

Comment:
let's make it clear in the description that "empty value or 'false' disables compression"

Reply:
How about turning this back into a boolean option and adding another configuration for compression type? This will clarify that the feature can now be toggled on or off with a boolean flag and introduce a new configuration option for compression type, ensuring users understand the change.

Reply:
yep sounds good

Reply:
Done